### PR TITLE
Specific curriculum ux

### PR DESCRIPTION
**Pull request by jaedynlee**
This PR includes:
* UX changes to an individual curriculum page to fit Celine's designs more closely.
* Removed the fonts we specified in custom CSS, because there were browser compatibility issues. They pretty much look the same as bootstrap default, so I didn't think this was a big deal.
* Balanced out the font weights a little bit.

Also:
* The materials section will look better once we have users enter materials one at a time.
* It looks like the 'subscribe to our newsletter' section should be a template or even part of base.html, so I didn't include that.
* The user I was testing with didn't have a name, apparently, which is why it shows up like that in my screenshots.

Screen Shot 2019-06-20 at 9 57 30 AM
With no reviews:
Screen Shot 2019-06-20 at 9 57 38 AM
With reviews:
Screen Shot 2019-06-20 at 10 04 38 AM

### CODE REVIEW

File: curriculum/templates/curriculum/lessonplan_detail.html [hunk 1]
```diff
@@ -14,142 +14,132 @@
...
+                        <span class="badge badge-outlined badge-secondary lessonplan-info">{% format_duration lessonplan.total_prep_time %} total prep time</span>
+                    </span>
+                    <span>
+                        <span class="sr-only">Works best for </span>
+                        <span class="badge badge-secondary badge-outlined lessonplan-info">{{ lessonplan.get_grade_level_display }}</span>
+                    </span>
+                    {% if lessonplan.web_only %}
+                    <span>
+                        <span class="sr-only">Chromebook compatible</span>
+                        <span>Chromebook compatible</span>
```
Comment (lukesanantonio) [inline curriculum/templates/curriculum/lessonplan_detail.html]: Did you mean for the Chromebook compatible to be it's own badge?

File: curriculum/templates/curriculum/lessonplan_detail.html [hunk 2]
```diff
@@ -14,142 +14,132 @@
...
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                </p>
+                <div class="col-md-8 pl-0">
+                    <p>{{ lessonplan.summary }}</p>
+                </div>
+                <div>
+                    <span>
+                        <span class="sr-only">In-class timeline: </span>
+                        <span class="badge badge-outlined badge-secondary lessonplan-info">{% number_to_words lessonplan.num_classes True %} {% format_duration lessonplan.single_class_time True %} class{{lessonplan.num_classes|pluralize:'es'}}</span>
```
Comment (lukesanantonio) [inline curriculum/templates/curriculum/lessonplan_detail.html]: Maybe we could add icons to make them easier to differentiate, what do you think?

File: curriculum/templates/curriculum/lessonplan_detail.html (hunk 1 above)
Comment (jaedynlee) [reply curriculum/templates/curriculum/lessonplan_detail.html]: Oh ha. I definitely didn't mean for it to look like that, but now I realize it's not specified in the wireframes. I think I will make it its own badge for now and @cyan478 can make that decision when she gets the chance.

File: curriculum/templates/curriculum/lessonplan_detail.html (hunk 2 above)
Comment (jaedynlee) [reply curriculum/templates/curriculum/lessonplan_detail.html]: A question for @cyan478

File: curriculum/templates/curriculum/lessonplan_detail.html [hunk 3]
```diff
@@ -14,142 +14,132 @@
...
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                </p>
-                            </div>
+                        <div class="col-lg-8 list-group-item px-0">
+                            <p class="mb-4 stars text-primary" data-average-rating="{{ fb.overall_rating }}" >
+                                <i class="fa-star" aria-hidden="true"></i>
```
Comment (kbusemeyer) [inline curriculum/templates/curriculum/lessonplan_detail.html]: I think the style might end up applied anyways from my PR, so might be unnecessary - but thoughts on changing the starts color to the gray background to make it consistent with the Find&Share Curricula?

Comment (jaedynlee) [reply curriculum/templates/curriculum/lessonplan_detail.html]: Awesome catch, they're gray in zeplin as well! Pushed the changes.

**Pull request merged**